apps/storybook/src/components/PillMenu.stories.tsx

```diff
@@ -99,28 +99,19 @@ export const Disabled: Story = {
   play: async () => {
```

Comment:
looks like UI test is still failing - let's just comment this out and re-visit later